client/packages/invoices/src/Returns/modals/InboundReturn/ReturnQuantitiesTable.tsx

```diff
@@ -15,35 +21,76 @@ export const QuantityReturnedTableComponent = ({
     line: Partial<GeneratedInboundReturnLineNode> & { id: string }
   ) => void;
 }) => {
+  // what to do here if there are multiple items? Some might have PV and others not...
+  // const { packVariantExists } = usePackVariant(item?.id || '', null);
+
   const columns = useColumns<GeneratedInboundReturnLineNode>(
     [
       'itemCode',
       'itemName',
       // 'itemUnit', // not implemented for now
       // 'location',
-      'batch',
-      'expiryDate',
       [
-        'numberOfPacks',
+        'batch',
         {
-          label: 'label.pack-quantity-issued',
           width: 125,
-          accessor: ({ rowData }) => rowData.numberOfPacksIssued,
+          accessor: ({ rowData }) => rowData.batch ?? '',
+          setter: updateLine,
+          Cell: TextInputCell,
+        },
+      ],
+      [
+        expiryInputColumn,
+        {
+          width: 150,
+          setter: l =>
+            updateLine({
+              ...l,
+              expiryDate: l.expiryDate
+                ? Formatter.naiveDate(new Date(l.expiryDate))
+                : null,
+            }),
+        },
+      ],
+      [
+        'packSize',
+        {
+          width: 100,
+          setter: updateLine,
+          Cell: PackSizeInputCell,
```

Comment:
I don't think pack size would be editable here, would it?

So it would be just a "NumberCell" rather than "NumberInputCell"?

Reply:
Oh I thought it would be? We're introducing new stock, it could have any pack size? Pack size is editable for Inbound Shipments, so just went with that...

Reply:
Yep editable, i guess there is a question should it be editable if we are adding fresh return from outbound line selection, but I think it's easier to just make it editable for both for now